### ISSUE TITLE
Test colour palette with minimalist UI

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/Design#150]
Test @kristinashu's new colour palette with one of your typical use cases. 

Use this palette: https://redpen.io/aq798f74c9a3ad7266

If you feel the need to make further adjustments to the colours, you can use @flukeout's tint generator here: http://flukeout.github.io/color-generator/ but please do document what changes you made and why.

Let me or @kristinashu know if you have any questions!

**Comment by vazquez** [comment]
I tested the new palette with some simple UI. Overall, these are my findings:
- The colour palette as a whole is really great. There's a lot of flexibility with them, and each colour stands apart from its sibling.
- The dark tones worked well for me 100% of the time. I used this dark tone for button borders, as an example.
- The `Lightness 40%` was a hit and miss. I found this value to work really well for warm values, and not so well for cool values in the blue spectrum.

Here is a Redpen with the palette explorations: https://redpen.io/p/at3be05cd4422c1d72

At the end, I added my proposed changes to the `Lightness 40%` values of:
- Mobile Blue
- Developer Blue
- Nightly Blue
- Aurora Purple
